ui/src/views/repositories/modals/remove-repository-modal/index.tsx

```diff
@@ -0,0 +1,80 @@
+import { ApolloError, useMutation } from '@apollo/client'
+import { Button, Modal, Toolbar } from '@mergestat/blocks'
+import { TrashIcon, XIcon } from '@mergestat/icons'
+import React, { useCallback } from 'react'
+import REMOVE_REPO from 'src/api-logic/graphql/mutations/remove-repo'
+import { useRepositoriesContext, useRepositoriesSetState } from 'src/state/contexts'
+import { showErrorAlert, showSuccessAlert } from 'src/utils/alerts'
+import useRepos from 'src/views/hooks/useRepos'
+
+export const RemoveRepositoryModal: React.FC = () => {
+  const [{ repoToRemove, search }] = useRepositoriesContext()
+  const { setShowRemoveRepositoryModal } = useRepositoriesSetState()
+  const { refetch } = useRepos(search)
+
+  const close = useCallback(() => {
+    setShowRemoveRepositoryModal(false)
+  }, [setShowRemoveRepositoryModal])
+
+  const [removeRepo] = useMutation(REMOVE_REPO, {
+    onError: (error: ApolloError) => {
+      showErrorAlert(error.message)
+    },
+    onCompleted: () => {
+      showSuccessAlert('Repository removed successfully')
+      refetch()
+      close()
+    }
+  })
+
+  return (
+    <Modal open onClose={close} size="sm">
+      <Modal.Header>
+        <Toolbar className="h-16 px-6">
+          <Toolbar.Left>
+            <Toolbar.Item>
+              <Modal.Title>Remove Repository</Modal.Title>
+            </Toolbar.Item>
+          </Toolbar.Left>
+          <Toolbar.Right>
+            <Toolbar.Item>
+              <Button
+                skin="borderless-muted"
+                onClick={close}
+                startIcon={<XIcon className="t-icon" />}
+              />
+            </Toolbar.Item>
+          </Toolbar.Right>
+        </Toolbar>
+      </Modal.Header>
+      <Modal.Body>
+        <div className="p-6">¿Are you sure to remove <b>{repoToRemove?.name}</b>?</div>
```

Comment:
The Spanish ¿ came out!

Reply:
```suggestion
        <div className="p-6">Are you sure to remove <b>{repoToRemove?.name}</b>?</div>
```

Reply:
One other request 😃 - if the repo has an `auto_import_id` associated with it (i.e. it came from a repo auto import) can we have an additional message here, in a `warning` callout that says "This repo was added by an automatic import, it will be re-added on the next run, unless the repo auto import is removed."

Reply:
Are you sure you want to make that suggestion?

Reply:
Done [bc53f39](https://github.com/mergestat/fuse/pull/256/commits/bc53f3927b43a41ca7584735026bf686c432609a)